app/views/govuk_publishing_components/components/docs/accordion.yml

```diff
@@ -185,15 +185,15 @@ examples:
     description: |
       Adds custom data attributes to each section of the accordion. Accepts a hash, so multiple attributes can be added.
 
-      The `data_attributes` hash is for the outermost element in the accordion.
+      The `data_attributes` option applies attributes to the outermost element in the accordion. Each item can also have a `data_attributes` hash, which are placed on the `button` that triggers the opening and closing - useful for differentiating between each section of the accordion.
 
-      Each item can also have a `data_attributes` hash. These `data_attributes` are placed on the `button` that triggers the opening and closing - useful for differentiating between each section of the accordion.
+      Data attributes can be added to the 'Show/hide all' link using the `data_attributes_show_all` option, primarily where custom tracking is required. These attributes are read from the accordion markup and then added to the link by JavaScript (which is how the link is created). More details on how this can be used with the GA4 click tracking can be found in the 'Advanced' section of the [click tracking documentation](https://github.com/alphagov/govuk_publishing_components/blob/main/docs/analytics-gtm/gtm-click-tracking.md).
```

Comment:
Not blocking but could link directly to the 'advanced' part of the docs https://github.com/alphagov/govuk_publishing_components/blob/main/docs/analytics-gtm/gtm-click-tracking.md#advanced-use

Reply:
Great suggestion. I did consider it, but was slightly cautious given the somewhat unfinished nature of a lot of this - thought it safer to link to the main file rather than a part of it inside that might change.